packages/typescript-estree/tests/lib/test-utils/expectToHaveParserServices.ts

```diff
@@ -0,0 +1,12 @@
+import type {
+  ParserServices,
+  ParserServicesWithTypeInformation,
+} from '../../../src';
+
+export function expectToHaveParserServices(
+  services: ParserServices | null | undefined,
+): asserts services is ParserServicesWithTypeInformation {
+  expect(services?.program).toBeDefined();
+  expect(services?.esTreeNodeToTSNodeMap).toBeDefined();
+  expect(services?.tsNodeToESTreeNodeMap).toBeDefined();
+}
```

Comment:
This overlaps heavily with `packages/parser/tests/tools/test-utils.ts` > `export function testServices`. Also the same file in `packages/type-utils`. Opportunity to deduplicate? Merge all into `type-utils`?

Reply:
TBH it feels smelly to export a util from our general-purpose utils package specifically so that we can delete 5 lines from one of our internal tests.
ESP when nobody externally would ever use the util in their tests.